contrib/golang/filters/http/source/go/pkg/http/config.go

```diff
@@ -55,7 +56,14 @@ func envoyGoFilterNewHttpPluginConfig(configPtr uint64, configLen uint64) uint64
 
 	configNum := atomic.AddUint64(&configNumGenerator, 1)
 	if httpFilterConfigParser != nil {
-		configCache.Store(configNum, httpFilterConfigParser.Parse(&any))
+		id, err := httpFilterConfigParser.Parse(&any)
```

Comment:
It's better to change the variable name `id --> parsedConfig`, as it does not return a typical ID.

Reply:
oh, nice catch, done, thanks.